README.md

```diff
@@ -123,6 +123,10 @@ and children.
   Accepts a GraphQLType instance and returns an array of field names.
   If not provided, a default behavior will be used.
 
+- `isKeyHandlerDisabled`: an optional boolean that disables key press event
```

Comment:
Definitely, I'll add this documentation. 

> How about disableRunQueryShortcut or disableExecuteQueryShortcut?

yeah, this sounds much batter. I think gave it more generic name just in case new shortcuts would be added in future. In this case, according my use-case, it does not really matter what shortcuts are disabled, as long as it covers all global shortcuts (added to a `document`). 

Just got one more idea: what do you think about `disableGlobalShortcuts`? I feel that it describes the use-case a bit better.